### ISSUE TITLE
Progress with DCLP problems

### ISSUE THREAD

**Issue by jcowey** [papyri/sosol#261]
I can now create a new DCLP record from advanced create. I have done so and submitted the entry. That means that #259 is sorted. Great. You closed the issue and all is good.

You probably know that one gets an error when trying to open in editor of a DCLP file which has related APIS info. An example that you tried yesterday:
https://papyri.info/dclp/60610
If I open in editor I get: 
![Bildschirmfoto 2021-01-14 um 11 42 06](https://user-images.githubusercontent.com/612992/104580419-906de300-565d-11eb-9cd7-8449dbd911c6.png)

You probably know this. Just wanted an issue marker for this one, as it is a blocker for people wanting to work on such records. And I know that some users have wanted to work on records like this.

**Comment by jcowey** [comment]
I see that in #217 you found a work around:

"a workaround is to use the editor's "Advanced Create" and paste the identifier e.g. papyri.info/apis/yale.apis.0010620000 into the PN ID field in the "From PN ID" section"

That did indeed function for me.

**Comment by ryanfb** [comment]
Yes, I believe this should be fixed once a full re-indexing is done with the latest code.

**Comment by jcowey** [comment]
OK thanks

**Comment by hcayless** [comment]
This appears to work, but the APIS record is not associated with the DCLP metadata and text in the editor, so something to do with the aggregation is broken.
